DemoQA/Tests/elements_webtables.js

```diff
@@ -221,4 +221,92 @@ test('Search table for department', async t => {
     const searchDepart =searchedUserDepartment[0].department;
 
     await t.expect(searchDepart).eql(randomUser.department);
-});
\ No newline at end of file
+});
+
+
+test('Change table row amount to 5', async t => {
+    await webTablesPage.navigateToWebTablesPage("Elements", "Web Tables");
+
+    const numberOfRowsToSelect = 5;
+
+    await webTablesPage.openPageSize();
```

Comment:
You have the selectPageSize function in the webTables page and you should use that in all tests and delete everything that is not necessary